runner/kubectl.go

```diff
@@ -63,10 +63,13 @@ func (k *Kubectl) Command(args []string) error {
 
 // NodeExists runs `sops exec-file KUBECONFIG 'kubectl get node NAME'` in the
 // background. If the node is not found a NodeNotFoundErr error is returned.
-func (k *Kubectl) NodeExists(name string) error {
+func (k *Kubectl) NodeExists(name string, addPrefix bool) error {
 	k.logger.Debug("kubectl_node_exists")
 
-	cmd := k.command("get", "node", k.fullNodeName(name))
+	if addPrefix == true {
```

Comment:
This can be simplified
```suggestion
	if addPrefix {
```